src/sinks/datadog/logs.rs

```diff
@@ -101,9 +102,9 @@ impl DatadogLogsConfig {
 
     fn batch_settings<T: Batch>(&self) -> Result<BatchSettings<T>, BatchError> {
         BatchSettings::default()
-            .bytes(bytesize::kib(100u64))
-            .events(20)
-            .timeout(1)
+            .bytes(bytesize::mib(5_u32))
+            .events(1_000)
+            .timeout(15)
```

Comment:
I wouldn't recommend increasing this one, at least not this much. All other "live" sinks leave the timeout at 1 second; only S3, GCS, and Azure Blob storage have it higher.

Reply:
Resolved in 05e662bb090de2c5ee983fba0b41c12e5a620409